src/main/java/tornadofx/CSS.kt

```diff
@@ -953,13 +952,69 @@ class CssSelectionBlock(op: CssSelectionBlock.() -> Unit) : PropertyHolder(), Se
     @Deprecated("Use and() instead as it's clearer", ReplaceWith("and(selector, *selectors, op = op)"))
     fun add(selector: Selectable, vararg selectors: Selectable, op: CssSelectionBlock.() -> Unit) = and(selector, *selectors, op = op)
 
-    fun and(selector: String, op: CssSelectionBlock.() -> Unit) = and(selector.toSelector(), op = op)
-    fun and(selector: Selectable, vararg selectors: Selectable, op: CssSelectionBlock.() -> Unit): CssSelection {
+    private fun addRelation(
+        relation: CssSubRule.Relation,
+        selector: Selectable, selectors: Array<out Selectable>,
+        op: CssSelectionBlock.() -> Unit
+    ): CssSelection {
         val s = select(selector, *selectors)(op)
-        selections[s] = true
+        selections[s] = relation
         return s
     }
 
+    /**
+     * [CssSubRule.Relation.REFINE]
+     */
+    fun and(selector: String, op: CssSelectionBlock.() -> Unit) = and(selector.toSelector(), op = op)
+
+    /**
+     * [CssSubRule.Relation.REFINE]
+     */
+    fun and(selector: Selectable, vararg selectors: Selectable, op: CssSelectionBlock.() -> Unit): CssSelection =
+        addRelation(CssSubRule.Relation.REFINE, selector, selectors, op = op)
+
+    /**
+     * [CssSubRule.Relation.CHILD]
+     */
+    fun child(selector: String, op: CssSelectionBlock.() -> Unit) = child(selector.toSelector(), op = op)
+
+    /**
+     * [CssSubRule.Relation.CHILD]
+     */
+    fun child(selector: Selectable, vararg selectors: Selectable, op: CssSelectionBlock.() -> Unit): CssSelection =
+        addRelation(CssSubRule.Relation.CHILD, selector, selectors, op = op)
+
+    /**
+     * [CssSubRule.Relation.DESCENDANT]
+     */
+    fun contains(selector: String, op: CssSelectionBlock.() -> Unit) = contains(selector.toSelector(), op = op)
+
+    /**
+     * [CssSubRule.Relation.DESCENDANT]
+     */
+    fun contains(selector: Selectable, vararg selectors: Selectable, op: CssSelectionBlock.() -> Unit): CssSelection =
```

Comment:
@edvin Do you think we should add the `contains` function? It is more complete, but it is also the default functionality if you don't use a function.

Reply:
Never mind, it's good for constancy.